openlibrary/macros/LoanStatus.html

```diff
@@ -38,10 +38,13 @@
   $:macros.ReadButton(ocaid, loan=user_loan, listen=True)
   $ return_url = page.url().rsplit('/', 1)[0] + '/do_return/borrow'
   <form action="$return_url" method="post" class="waitinglist-form return-book">
+    <p>
```

Comment:
Why is <p> tag even required?

Reply:
The above read cta_btn_grp occurs in pairs and forms a segment of their own. Any next button(ret in this case) that comes immediately next to it collides into it based on the available space and missing line. So even if something that looks fine in Desktop version(a) looks collided in the mobile version(b).

![a](https://user-images.githubusercontent.com/23471306/76710253-f763f800-672b-11ea-90d2-cd4fa4c37776.png)
![b](https://user-images.githubusercontent.com/23471306/76710254-f8952500-672b-11ea-9430-50159afedfeb.png)